libs/ledgerjs/packages/cryptoassets/src/currencies.ts

```diff
@@ -3614,6 +3614,7 @@ export const cryptocurrenciesById: Record<CryptoCurrencyId, CryptoCurrency> = {
         uri: "https://api.etherscan.io",
         type: "etherscan",
       },
+      gasTracker: { uri: "", type: "ledger" },
```

Comment:
Please add this as well for `polygon_as_evm_test_only" 🙏

Reply:
Done here a4ce505d893dd6ebdd7b1b9753f883029d3d1306